exercises/zipper/canonical-data.json

```diff
@@ -279,6 +279,51 @@
         "value": 3
       }
     },
+    {
+      "uuid": "b9aa8d54-07b7-4bfd-ab6b-7ff7f35930b6",
+      "description": "left, right, up, and up",
```

Comment:
Would be good for the description to say what aspect is being tested, rather than just list the operations.

(I realise that _some_ other test descriptions in this file follow the same format, but no reason to continue a less clear approach)

Reply:
Good idea. Done.